config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  # get 'menu/index'
```

Comment:
you can remove this, old syntax

Reply:
never push up commented out code if you are not in the middle of implementing a feature

Reply:
Comment cleaned and code refactored to modern syntax.